### DISCUSSION TITLE
Cannot find where to use this sonar feature...

### DISCUSSION THREAD

**Discussion by kenherring-rkt** [Riverside-Software/sonar-openedge#990] [Q&A] [answered]
In the [wiki under "features"](https://github.com/Riverside-Software/sonar-openedge/wiki/Features) there is this screenshot.  How do I find this in my own sonar instance?  I seem to only be able to see graphs under "Activity" and "Measures".  I'm sure I'm just failing to look in the right place, but hoping someone can help point me in the right direction.

![image](https://user-images.githubusercontent.com/87667618/187545949-61abfc72-b692-4d3c-836b-bb803152c0fd.png)

Thanks!

**Comment by gquerret** [comment, marked as answer]
This kind of specific screen has been deprecated in SonarQube in version 8 IIRC, so it's not available anymore. The wiki page will be updated.
As you said in your comment, those metrics are available in the Measures screen. Values available on the left side.

<img width="288" alt="image" src="https://user-images.githubusercontent.com/6498723/187670918-866a0e18-32cf-4b90-b906-2a88cc80d360.png">

**Reply by kenherring-rkt** [reply]
Thanks for the quick response.  The old screenshot looks so much cooler than the new one, which is exactly what I see now. Thanks!
